_translator-files/po/fr/Include-Shared-Commands.po

```diff
@@ -2,11 +2,12 @@
 # Copyright (C) YEAR Free Software Foundation, Inc.
 # This file is distributed under the same license as the PACKAGE package.
 # Olivier Humbert <please@seeGitHubOrWeblate.local>, 2022.
+# Olivier Humbert <trebmuh@tuxfamily.org>, 2022.
```

Comment:
Duplicate needs to be removed

Reply:
It seems to be something which need to be removed in the source code here on github since I can't managed to do that through the weblate interface.

Reply:
Yes. I think so too